Automattic/jetpack
# Boost: Fix overall speed scores info tooltip UI

The overal speed score info tooltip is being cut off as shown in the screenshot below:

<img width="1536" alt="Screen Shot 2021-10-07 at 2 23 21 PM" src="https://user-images.githubusercontent.com/927932/136325558-56b37d15-7191-4746-b4cf-f58bc4a08197.png">


#### Changes proposed in this Pull Request:

* Refactor the CSS overflow rules blocking the display of the overall speed score info tooltip. Move the rule to the parent container and force it on the x-axis only.

#### Jetpack product discussion
N/A

#### Does this pull request change what data or activity we track or use?
N/A

#### Testing instructions:
- Generate a score, click on the info tooltip icon next to the overall score and see if the tooltip is shown entirely as the screenshot below:
<img width="1536" alt="Screen Shot 2021-10-07 at 2 24 00 PM" src="https://user-images.githubusercontent.com/927932/136325469-02dd849e-e71e-42fc-abb4-e6ce374c14b4.png">
- Also make sure that it works well with the rating popin and when closing that popin it does not create an overflow/scroll on the right hand side of the screen

See final demo on how it should behave below:

https://user-images.githubusercontent.com/927932/136330449-e3afb44b-99f0-49ee-a6cd-8e3795d85779.mp4
